### PR TITLE
A lot of things : restriction on functions, a lot of margin and padding modification for optimization.. (see commits comment)

### PR DESCRIPTION
**Pull request by CharleneDR**
To save time, I have merged with the US_patch branch as only a h2 tag was wrong on the PR (on line 9 of src/View/User/login.html.twig).
So, this PR contains Ibrahim's commits (that you've already checked) and mine.

 I know what you'll say about this but, I try..

### CODE REVIEW

File: src/View/Contact/contact.html.twig [hunk 1]
```diff
@@ -4,16 +4,14 @@
...
-			<h2 class="text-center">Contactez nous</h2>
-
-			<form method="post" class='form m-3 bg-lightgray p-3 shadow col-sm-10 col-md-8 col-lg-6 mx-auto' accept-charset="UTF-8">
+		<div class='m-3 mb-4'>
+			<form method="post" class='form m-3 bg-lightgray p-3 shadow col-sm-10 col-md-8 col-lg-6 mx-auto' accept-charset="UTF-8" action="#ancre">
 				<input type="hidden" value="{{ user.id }}" name="id">
 
 				<div class='container'>
-					<div class="row">
+					<div class="row" id='ancre'>
```
Comment (aldaitzwild) [inline src/View/Contact/contact.html.twig:14]: It should be in english

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Contact/contact.html.twig (hunk 1 above)
Comment (CharleneDR) [reply src/View/Contact/contact.html.twig]: done

File: src/View/Game/_gamecard.html.twig [hunk 2]
```diff
@@ -29,7 +29,10 @@
 			</p>
 		</div>
 		<div class="card-footer text-center w-100 d-flex justify-content-around">
-			{% if game.availability == true %}
+			{% if game.availability == true and game.game_id in requestedGames == true %}
```
Comment (aldaitzwild) [inline src/View/Game/_gamecard.html.twig:32]: the `== true` is not necessary

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Game/_gamecard.html.twig (hunk 2 above)
Comment (CharleneDR) [reply src/View/Game/_gamecard.html.twig:32]: I changed it. But i think it's more reader-friendly to write it, no ?

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**